docs/docusaurus/docs/cloud/connect/connect_snowflake.md

```diff
@@ -0,0 +1,213 @@
+
+2. In the **User access tokens** pane, click **Create user access token**.
+
+3. Complete the following fields:
+
+   - **Token name** - Enter a name for the token that will help you quickly identify it.
+
+   - **Role** - Select **Admin**. For more information about the available roles, click **Information** (?).
+
+4. Click **Create**.
+
+5. Copy and then paste the user access token into a temporary file. The token can't be retrieved after you close the dialog.
+
+6. Click **Close**.
+
+7. Copy the value in the **Organization ID** field into the temporary file with your access token and then save the file. 
+
+   GX recommends deleting the temporary file after you set the environment variables.
+
+## Deploy the GX Agent
+
+You deploy and run the GX Agent within your production cloud services environment. You can deploy the GX Agent container in any environment where you can run Docker container images or create Kubernetes clusters.
+
+<Tabs
+  groupId="connect-snowflake"
+  defaultValue='docker'
+  values={[
+  {label: 'Docker', value:'docker'},
+  {label: 'Kubernetes', value:'kubernetes'},
+  ]}>
+<TabItem value="docker">
+
+To learn how to deploy a Docker container image in a specific environment, see the following documentation:
+
+- [Quickstart: Deploy a container instance in Azure using the Azure CLI](https://learn.microsoft.com/en-us/azure/container-instances/container-instances-quickstart)
+
+- [Build and push a Docker image with Google Cloud Build](https://cloud.google.com/build/docs/build-push-docker-image)
+
+- [Deploy Docker Containers on Amazon ECS](https://aws.amazon.com/getting-started/hands-on/deploy-docker-containers/)
+
+
+1. Download the GX Agent Docker container image from [Docker Hub](https://hub.docker.com/r/greatexpectations/agent).
+
+2. After configuring your cloud service to run Docker containers, run the following Docker command to start the GX Agent: 
+
+   ```bash title="Terminal input"
+   docker run -it \
+   -e GX_CLOUD_ACCESS_TOKEN= YOUR_ACCESS_TOKEN \ 
+   -e GX_CLOUD_ORGANIZATION_ID= YOUR_ORGANIZATION_ID \ 
+   -e GX_CLOUD_SNOWFLAKE_PW= YOUR_SNOWFLAKE_PASSWORD \ 
+   greatexpectations/agent:latest
+    ```
+    Replace `YOUR_ACCESS_TOKEN` and `YOUR_ORGANIZATION_ID` with the values you copied previously and `YOUR_SNOWFLAKE_PASSWORD` with your Snowflake password. The `YOUR_SNOWFLAKE_PASSWORD` variable is optional, and you can remove it if you prefer to provide the password in GX Cloud.
+
+3. Run the following command to use the GX Agent image as the base image and optionally add custom commands:
+
+   ```bash title="Terminal input"
+   FROM greatexpectations/agent
+   RUN echo "custom_commands"
+   ```
+4. Run the following command to rebuild the Docker image:
+
+   ```bash title="Terminal input"
+   docker build -t myorg/agent
+   ```
+
+</TabItem>
+<TabItem value="kubernetes">
+
+You can deploy the GX Agent in any environment you create Kubernetes clusters. For example:
```

Comment:
It is "You can" for Kubernetes and "You" for Docker. Do we want these to be consistent?

Reply:
@Shinnnyshinshin, the first instance (you deploy) is a direct statement. There is no ambiguity. The GX Agent must be deployed in a production environment. With the second instance (you can deploy), several choices are available to a user. Therefore, the verb _can_ is applicable because a user _**can**_ choose where the GX Agent is deployed. It's a subtle distinction.